samcli/commands/local/lib/sam_base_provider.py

```diff
@@ -3,17 +3,148 @@
 """
 
 from samcli.lib.samlib.wrapper import SamTranslatorWrapper
+from samtranslator.intrinsics.resolver import IntrinsicsResolver
+from samtranslator.intrinsics.actions import RefAction
 
 
 class SamBaseProvider(object):
     """
     Base class for SAM Template providers
     """
 
+    _DEFAULT_PSEUDO_PARAM_VALUES = {
+        "AWS::AccountId": "123456789012",
+        "AWS::Partition": "aws",
+
+        # There is not much value in inferring actual AWS region here. These values are just placeholders to help
+        # local testing and not representative of any environment.
+        "AWS::Region": "us-east-1",
```

Comment:
Region is possible. Partition is harder because it requires us to maintain a list of region:partition mapping.

I don't want to pass region to the SAM Provider because it doesn't make sense. I will try to find another solution

Reply:
StackId is rarely used and partition is usually "aws". So I am going to skip those.